github/repos_lfs.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
```

Comment:
For new files, we give them the year that they are created.

```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```